shared/js/otSdkDriver.js

```diff
@@ -622,6 +624,7 @@ loop.OTSdkDriver = (function() {
         srcMediaElement: sdkSubscriberVideo
       }));
 
+      this._notifyMetricsEvent("Session.subscribeCompleted");
```

Comment:
It's not obvious why this wants to be logged here rather than at the beginning of the function, since things could blow up before we get here.  Adding a comment about that might be helpful.